pkg/flowaggregator/flowaggregator.go

```diff
@@ -252,17 +253,29 @@ func (fa *flowAggregator) initExportingProcess() error {
 		return fmt.Errorf("got error when initializing IPFIX exporting process: %v", err)
 	}
 	fa.exportingProcess = ep
-	fa.templateID = fa.exportingProcess.NewTemplateID()
-	templateSet := ipfix.NewSet(ipfixentities.Template, fa.templateID, false)
 
-	bytesSent, err := fa.sendTemplateSet(templateSet)
+	fa.templateIDv4 = fa.exportingProcess.NewTemplateID()
```

Comment:
we send both templates regardless of the IP family / families supported by the cluster? This seems fine but maybe we can add a comment.